services/brig/src/Brig/API.hs

```diff
@@ -1377,16 +1383,26 @@ addBlacklist emailOrPhone = do
 
 canBeDeleted :: UserId ::: TeamId -> Handler Response
 canBeDeleted (uid ::: tid) = do
-    onlyOwner <- lift (Team.isOnlyTeamOwner uid tid)
+    onlyOwner <- lift (Team.teamOwnershipStatus uid tid)
     case onlyOwner of
        Team.IsOnlyTeamOwner       -> throwStd noOtherOwner
        Team.IsOneOfManyTeamOwners -> pure ()
        Team.IsNotTeamOwner        -> pure ()
-       Team.NoTeamOwnersAreLeft   -> do
+       Team.NoTeamOwnersAreLeft   -> do  -- (keeping the user won't help in this case)
            Log.warn $ Log.field "user" (toByteString uid)
                     . Log.msg (Log.val "Team.NoTeamOwnersAreLeft")
     return empty
 
+isTeamOwner :: UserId ::: TeamId -> Handler Response
```

Comment:
Is this really necessary? This simply looks up in galley and checks the status; or was it added here because atm spar talks only to brig?

Reply:
I think `getZUsrTeam` returns a `teamid` even if the user is only member, not owner.  But you are correct if you mean that we should probably make that `getZUserOwnedTeam` and simplify the code accordingly.  I'll do that immediately!

Reply:
b344ade720d06f9f9034abb42e3c9462072304d8